app/library/Middleware/ViewMiddleware.php

```diff
@@ -31,24 +31,28 @@ public function call(Micro $application)
         $registry     = $application->registry;
         $viewName     = $registry->view;
 
-        if ('production' === $application->config->get('env')) {
+        if ('production' === $application->config->get('app')->get('env')) {
             $application->viewSimple->cache(['key' => $cacheKey]);
         }
 
-        $application->viewSimple->setVars(
-            [
-                'page'          => $registry->slug,
-                'language'      => $registry->language,
-                'imageLanguage' => $registry->imageLanguage,
-                'contributors'  => $registry->contributors,
-                'languages'     => $registry->menuLanguages,
-                'noindex'       => $registry->noindex,
-                'releases'      => $registry->releases,
-                'version'       => $registry->version,
-            ]
-        );
-
-        $contents = $application->viewSimple->render($viewName);
+        if (true === $application->viewCache->exists($cacheKey)) {
```

Comment:
this method return true or false so `===` not needed at all.
just `if ($application->viewCache->exists($cacheKey)) {`